webapp/channels/src/components/channel_settings_modal/__snapshots__/channel_settings_access_rules_tab.test.tsx.snap

```diff
@@ -24,6 +24,30 @@ exports[`components/channel_settings_modal/ChannelSettingsAccessRulesTab should
     >
       Select attributes and values that users must match in addition to access this channel. All selected attributes are required.
     </p>
+    <div
+      class="ChannelSettingsModal__autoSyncSection"
+    >
+      <label
+        class="ChannelSettingsModal__autoSyncLabel"
+      >
+        <input
+          class="ChannelSettingsModal__autoSyncCheckbox"
+          id="autoSyncMembersCheckbox"
+          name="autoSyncMembers"
+          type="checkbox"
+        />
+        <span
+          class="ChannelSettingsModal__autoSyncText"
+        >
+          Auto-add members based on access rules
```

Comment:
```suggestion
          Auto-add members based on access control policies
```

Reply:
there is another PR where everything is renamed correctly to access control https://github.com/mattermost/mattermost/pull/33810